### PR TITLE
1.7.6 release

### PR DESCRIPTION
**Pull request by jordimassaguerpla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,20 @@ In case you want to use it without SUSE Manager:
...
+https://github.com/SUSE/smdba/commit/530731003cbf3a275f7fe3bfb97b4337f975a38e
+
+* After commiting it to the master branch, create a tag and push it:
+ ```git tag X.Y.Z && git push --tags``` where X.Y.Z is the new version
+
+* Call build-src.sh to create the tarball
+
+* Update the tarball in:
+
+https://build.opensuse.org/package/show/systemsmanagement:Uyuni:Stable/smdba
```
Comment (juliogonzalez) [inline README.md:43]: Wrong project. Submissions go to Master. Things into Stable from Master when we promote for a release.

What about changing the SPEC version and updating the changelog?

Also we should mention that backporting to Devel:Galaxy:Manager:Head is required, and most likely to released versions of SUSE Manager.

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
Some comments about the releasing procedure.

File: README.md [hunk 2]
```diff
@@ -27,3 +27,22 @@ In case you want to use it without SUSE Manager:
...
+
+* Update version in setup.py and src/smdba/smdba (as an example see
+https://github.com/SUSE/smdba/commit/530731003cbf3a275f7fe3bfb97b4337f975a38e
+
+* After commiting it to the master branch, create a tag and push it:
+ ```git tag X.Y.Z && git push --tags``` where X.Y.Z is the new version
+
+* Call build-src.sh to create the tarball
+
+* Update the tarball, spec file and changelog at:
```
Comment (juliogonzalez) [inline README.md:41]: Is the SPEC and changelog file in this project? I'd say it's not, and if so, you can't upload them. You need to edit them.

And only now I remembered: you need to create a SR to systemsmanagement:Uyuni:Master/smdba, so you don't upload directly.

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jordimassaguerpla) [reply README.md:41]: It states "update" not "upload" ;) let me make it more clear

Comment (juliogonzalez) [reply README.md]: True, my mistake. Still that could make some people think that `osc ci` is just fine :-D

File: README.md [hunk 3]
```diff
@@ -27,3 +27,24 @@ In case you want to use it without SUSE Manager:
...
+
+* Update version in setup.py and src/smdba/smdba (as an example see
+https://github.com/SUSE/smdba/commit/530731003cbf3a275f7fe3bfb97b4337f975a38e
+
+* After commiting it to the master branch, create a tag and push it:
+ ```git tag X.Y.Z && git push --tags``` where X.Y.Z is the new version
+
+* Call build-src.sh to create the tarball
+
+* Upload the tarball, and update the spec file and changelog at:
```
Comment (juliogonzalez) [inline README.md:41]: This still says `upload the tarball at https://build.opensuse.org/package/show/systemsmanagement:Uyuni:Master/smdba`.

I suggest:

> * You need to update https://build.opensuse.org/package/show/systemsmanagement:Uyuni:Master/smdba, but **do not submit directly with a commit**.
> * Create a Submit Request that replaces the tarball and updates the SPEC and the changelog (see https://openbuildservice.org/help/manuals/obs-user-guide/art.obs.bg.html#sec.obsbg.uc.branchprj for details)
> * Prepare Submit Requests to to SUSE Manager Head and probably the released versions
>    Tip: if for the Submit Request for Uyuni you told OBS not to clean the branched package when accepted, you can use that branched package for the submissions to SUSE Manager as well)

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

**Pull request merged**